app/controllers/api/application_controller.rb

```diff
@@ -1,11 +1,15 @@
 module Api
   class ApplicationController < ::ActionController::API
-    before_filter :authenticate!
+    before_action :authenticate!
 
     protected
 
     def authenticate!
-      render json: { error: 'Unauthorized' }, status: 401 unless api_token
+      render json: { error: 'Unathorized' }, status: 401 unless current_user
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.